### PR TITLE
Add List of Instructors for each Position

### PR DESCRIPTION
**Pull request by michellemtchai**
- fix syntax error in `mock_chass.json`
- add `instructors` attribute to each `Instructor` object in JSON
- updated API documentation to include `instructors` attribute
- added `/instructors` route
  - `/instructors` gets all instructors in the system
  - `/instructors/:id` get an instructor by its `id`
    - it returns `status: 404` if the `Instructor` with `id` doesn't exist
- added `instructors` as an editable attribute in `PATCH /positions/:id`
- remove `app/models/course.rb`
- added API documentation for `instructors` route

### CODE REVIEW

File: app/controllers/positions_controller.rb [hunk 1]
```diff
@@ -2,18 +2,24 @@ class PositionsController < ApplicationController
...
-    render json: position.to_json
+    position = Position.includes(:instructors).find(params[:id])
+    render json: position.to_json(include: [:instructors])
   end
 
   def update
     position = Position.find(params[:id])
     position.update_attributes!(position_params)
+    if params[:instructors]
+      puts 'not null'
```
Comment (freeatnet) [inline app/controllers/positions_controller.rb]: debug output?

File: app/controllers/positions_controller.rb [hunk 2]
```diff
@@ -2,18 +2,24 @@ class PositionsController < ApplicationController
...
+      position.instructor_ids = JSON.parse params[:instructors]
```
Comment (freeatnet) [inline app/controllers/positions_controller.rb]: Let's see if we can use an array here instead.
If requesting with `application/x-www-form-urlencoded`, we can do `instructors[]=2&instructors[]=4` to make an array; or we can use JSON body request, `{"instructors": [2,4]}`

File: app/controllers/positions_controller.rb (hunk 1 above)
Comment (michellemtchai) [reply app/controllers/positions_controller.rb]: Sorry. I was going to add more commits to this branch. I'll fix that.

File: app/controllers/positions_controller.rb (hunk 2 above)
Comment (michellemtchai) [reply app/controllers/positions_controller.rb]: Okay. I'll try that.

Comment (michellemtchai) [reply app/controllers/positions_controller.rb]: The `application/x-www-form-urlencoded` worked. The JSON body request doesn't seem to work. (Well, at least with Postman).

**Review by freeatnet — APPROVED** [review]
Looks good!

**Pull request merged**